modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/OutputWriter.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+using System.Text;
+using Xunit.Abstractions;
+
+namespace Godot.SourceGenerators.Tests;
+
+public class OutputWriter : TextWriter
```

Comment:
Not sure why this class is needed, doesn't xUnit output to the console by default?

Reply:
This was grab messages printed within the generators themselves during debug. It's going away.